### DISCUSSION TITLE
The CPU utilization rate of fullnode process often exceeds 100%.pls help me ！

### DISCUSSION THREAD

**Discussion by ben00007** [Chia-Network/chia-blockchain#5108] [Support]
The CPU utilization rate of fullnode process often exceeds 100%. Will this one affect the profit of mining? If there are 1300 hosts to solo mining, please confirm whether this scheme is feasible. A full node connects 13 farmers, and each farmer has 100 machines. Is this scheme feasible? How will the government recommend the deployment? Thank you ！

**Comment by cccat6** [comment]
You might need to upgrade your CPU.
In my practice, except harvester, all the other components including full_node and wallet are single-threaded. They will eat up a thread in your machine when they are syncing. It usually will drop down very quickly unless you are consistently syncing. Once the sync down, it should not exceed 100% as usual.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
